docker/docker-tezos-packages.py

```diff
@@ -159,6 +168,23 @@ def build_packages(pkgs, distros):
         print("Unrecoverable error occured.")
         sys.exit(exit_code)
 
+    artifacts = (os.path.join(args.output_dir, x) for x in os.listdir(args.output_dir))
+    if args.gpg_sign and args.type == "source":
+        if target_os == "ubuntu":
+            for f in files:
+                if f.endswith(".changes"):
+                    call(
+                        f"sed -i 's/^Changed-By: .*$/Changed-By: {args.gpg_sign}/' {f}"
```

Comment:
This is also done in `scripts/publish-native-packages.sh`, I suppose that now it could be removed from there and CI updated to instead using this option, no?

Moreover, there are also instructions in `docker/README.md` that mention running this command. That doc should also be updated now IMO

Reply:
> I suppose that now it could be removed from there and CI updated to instead using this option, no?